api/src/config.rs

```diff
@@ -1,13 +1,18 @@
 use jwt_simple::algorithms::HS256Key;
 use std::net::IpAddr;
 
+use base64::Engine;
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
 
 pub(crate) struct Base64(pub(crate) HS256Key);
 
 impl std::fmt::Debug for Base64 {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, r#"b64"{}""#, &base64::encode(self.0.to_bytes()))
+        write!(
+            f,
+            r#"b64"{}""#,
+            &base64::engine::general_purpose::STANDARD_NO_PAD.encode(self.0.to_bytes())
```

Comment:
Why STANDARD_NO_PAD instead of STANDARD? I think the standard `base64` unix util uses padding, this would allow to use it to populate this value

Reply:
My bad. Thought padding with `=` might not be useful for a secret string ?
I should change it though

Reply:
It's a bit better to be compatible with the `base64` util I think indeed